.github/scripts/release.py

```diff
@@ -359,6 +359,15 @@ def autoRelease(self):
         # Update the header in each c/assembly file
         self.updateFileHeaderVersions(['FreeRTOS Kernel V','FreeRTOS Kernel <DEVELOPMENT BRANCH>'], 'FreeRTOS Kernel V%s' % self.version)
 
+
+    def autoRelease(self):
+        info('Auto-releasing FreeRTOS Kernel V%s' % self.version)
```

Comment:
Nit: if `no_not_update_src` is false you will log the same info statement twice.